core/services/chainlink/application.go

```diff
@@ -179,6 +179,7 @@ type ApplicationOpts struct {
 	LoopRegistry               *plugins.LoopRegistry
 	GRPCOpts                   loop.GRPCOpts
 	MercuryPool                wsrpc.Pool
+	CapabilitiesRegistry       *capabilities.Registry
```

Comment:
Please use `coretypes.CapabilitiesRegistry` here rather than the concrete type (`chainlink-common/pkg/types/core`) as that will make loopification easy